syntax = "proto3";

package graph_pb;

message Edge {
    int32  from               = 1;
    int32  to                 = 2;
    double weight             = 3;
    int32  span_count         = 4;
    string stop_or_route_name = 5;
}

message VertexToEdgeIds {
    repeated int32 edge_ids = 1;
}

message Graph {
    repeated Edge            edges    = 1;
    repeated VertexToEdgeIds vertexes = 2;
}